src/components/SlotSelectorModal/slotVisibilities.ts

```diff
@@ -0,0 +1,453 @@
+import { TemplateName } from "../../api";
+
+type VisibilityLevel = "common" | "occasional";
+
+interface SlotVisibilities
+  extends Partial<Record<TemplateName, VisibilityLevel>> {
+  _default?: VisibilityLevel;
+}
+
+/**
+ * This object encodes which slots are commonly, occasionally, or uncommonly measured at sample
+ * collection time. This was the output of a manual curation process. At some point we might
+ * consider encoding this directly in the submission schema somehow. But for now, this is a simple
+ * way to get the job done. The structure of the object is:
```

Comment:
```ts
/**
 * [...] But for now, this is a simple
 * way to get the job done. [...]
 */
```

I'm OK with this file being merged in as is, with that framing of the situation.

One thing that came to mind while looking at the `slotVisibilities` object below was that those three strings could be implemented as [Enum](https://www.typescriptlang.org/docs/handbook/enums.html) values rather than literal strings, which would allow for the inner objects to look like this:

```diff
  agrochem_addition: {
-    _default: "occasional",
+    _default: Frequency.Occasional,
  },
```

That might complicate whatever automatic process was used to convert the Google Sheets document into this object, though.

Reply:
> whatever automatic process was used to convert the Google Sheets document into this object

Haha, if only. No, it was a manual process. So no worries (or lots of worries??) there. 

The values in this object are already constrained via a string union type: https://github.com/microbiomedata/nmdc-field-notes/blob/b56760aba47238449d06859107484939a57e2c67/src/components/SlotSelectorModal/slotVisibilities.ts#L3

I couldn't tell you in detail the pros/cons of string unions vs enums in a situation like this. I guess I lean toward the string unions as being a bit more lightweight.